xml_inscriptions/transcribed/MI.AA.UM.KM.L.892.xml

```diff
@@ -43,7 +43,10 @@
                               <depth>0.15</depth>
                            </dimensions>
                         </support>
-                        <condition><!--Add ana attribute with controlled value for condition--><p>broken into seventeen fragments, one of which is missing from the bottom edge, probably at the point of impact of a pick</p>
+                        <condition ana="#complete.broken">
```

Comment:
We might actually label this contiguous fragments rather than complete.broken; worth double-checking.